app/Services/Utils/Facades/EmailExcerpt.php

```diff
@@ -0,0 +1,27 @@
+<?php namespace App\Services\Utils\Facades;
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use Illuminate\Support\Facades\Facade;
+
+/**
+ * Class EmailExcerpt
+ * @package services\utils\Facades
+ */
+class EmailExcerpt extends Facade
```

Comment:
where are u binding this facade to container ?
check https://github.com/OpenStackweb/summit-api/blob/ade98172ad7563b781cbaa75eda06d5fd03d4d87/app/Providers/AppServiceProvider.php#L631